### PR TITLE
Rephrase first sentence in manage-hybrid-exchange-recipients-with-management-tools.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/3081.

### CODE REVIEW

**Comment by dariomws** [comment]
@chrisda @JoanneHendrickson, I'll appreciate your review.

@JohanFreelancer9, I'll appreciate your feedback about wording and grammar.

File: Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ manager: serdars
...
+Previously, in Exchange Hybrid environments, after you move all your mailboxes to Exchange Online, you had to have an active Exchange Server to manage recipients' attributes. First, attributes were edited using an Exchange Server in on-premises Active Directory, which were then copied to Azure AD using directory synchronization. On-premises recipients can't be modified directly in Azure AD or Exchange Online. Therefore you had to keep an Exchange Server running using directory synchronization via the Azure AD Connect tool. For more information, see [Why you may not want to decommission Exchange servers from on-premises](/exchange/decommission-on-premises-exchange#why-you-may-not-want-to-decommission-exchange-servers-from-on-premises).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md:22]:
```suggestion
Previously, in Exchange Hybrid environments, after you have moved all your mailboxes to Exchange Online, you had to have an active Exchange Server to manage recipients' attributes. First, attributes were edited using an Exchange Server in on-premises Active Directory, which was then copied to Azure AD using directory synchronization. On-premises recipients can't be modified directly in Azure AD or Exchange Online. Therefore you had to keep an Exchange Server running using directory synchronization via the Azure AD Connect tool. For more information, see [Why you may not want to decommission Exchange servers from on-premises](/exchange/decommission-on-premises-exchange#why-you-may-not-want-to-decommission-exchange-servers-from-on-premises).
```

Comment (dariomws) [reply Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md:22]: Agree with "have moved" but not sure about "was", we are talking about the attributes so it should be "were".

Comment (dariomws) [reply Exchange/ExchangeHybrid/manage-hybrid-exchange-recipients-with-management-tools.md]: Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**